### DISCUSSION TITLE
Cicero co-accessibility values

### DISCUSSION THREAD

**Discussion by coopershawna** [stuart-lab/signac#853] [Q&A] [answered]
Is there a way to get the numerical values for coaccessibility scores plotted in "Links"?
Thanks

**Comment by timoast** [comment, marked as answer]
Yes, you can use the `Links()` function to get the link granges object, along with the scores

**Reply by coopershawna** [reply]
Thank you. I appreciate the help!
Curious, is there a way to force the scale on the links plot so that separate plots are easier to visually compare?

**Reply by timoast** [reply]
Currently there's no way to manually set the max value for the colorscale for links

**Reply by coopershawna** [reply]
Thank you!
